README.md

```diff
@@ -28,6 +28,7 @@ Usage: java org.plumelib.javadoc.RequireJavadoc [options] [directory-or-file ...
   --exclude=<regex>                - Don't check files or directories whose pathname matches the regex
   --dont-require=<regex>           - Don't report problems in Java elements whose name matches the regex
   --dont-require-private=<boolean> - Don't report problems in elements with private access [default: false]
+  --dont-require-properties=<boolean> - Don't report problems about trivial getters and setters [default: false]
```

Comment:
Should the value be verbose `--dont-require-trivial-properties` so that there could be a `--dont-require-properties` use case is also available?